ntbs-service/Pages/Alerts/TransferRequestAction.cshtml.cs

```diff
@@ -148,10 +148,16 @@ public async Task AcceptTransferAndDismissAlertAsync()
             Notification.HospitalDetails.TBServiceCode = TransferAlert.TbServiceCode;
             Notification.HospitalDetails.HospitalId = TargetHospitalId;
             Notification.HospitalDetails.CaseManagerId = TargetCaseManagerId;
+
+            // Drop consultant and local patient ID because these are almost certainly going to be different at the new
+            // TB service, as per NTBS-1559
+            Notification.HospitalDetails.Consultant = null;
+            Notification.PatientDetails.LocalPatientId = null;
             // We want to save all these changes in the same transaction so that we make sure the time between the first
             // and last audit logs are within the timeframe for them to be grouped together on the changes page. We
             // experienced an issue where the audits from an acceptance were over this threshold and our app didn't
             // know how to handle the group. See NTBS-2457
+            Service.UpdatePatientDetailsWithoutSaveOrSideEffects(Notification, Notification.PatientDetails);
             Service.UpdateHospitalDetailsWithoutSave(Notification, Notification.HospitalDetails);

```

Comment:
The naming of these methods is inconsistent, despite the fact that they are conceptually very similar. I think `WithoutSave` is sufficient here - and I don't think "without side effects" is strictly accurate, since the Notification is mutated in place by this method.

Reply:
Yeah I was a bit unsure what to call this. What I was meaning here was that I'm not running any of the other side effects compared to `UpdatePatientDetails`, like `UpdatePatientFlagsAsync` which formats NHS numbers, years of entry, etc. and `AutoDismissAlertAsync<DataQualityBirthCountryAlert>`, I'm just pushing the changes to the local patient ID in this case. If you think `WithoutSave` gets this across fine then I'm v happy to change it

Reply:
I think `WithoutSave` is fine here.

There's definitely an argument that the original `UpdatePatientDetails` ought to have a clearer name though... maybe something along the lines of `ValidateAndSavePatientDetailsAndDismissAlerts`... but given that there are lots of methods that are like it I think leaving it alone may be the best option.

Reply:
Yeah that makes sense, I've renamed it